test/support/single.spec.js

```diff
@@ -1,10 +1,7 @@
 require('./../utils/starter.spec.js').runTests({
     description: 'In reference,',
     testCases: [
-        "/reference/tests/01.reference.js",
```

Comment:
Yes, I disabled other tests during development. This file is used when we are doing development to run a single spec file. Updating or removing testcases from this file doesn't hamper running all other test-cases as they're part of some other configuration.

https://github.com/informatics-isi-edu/ermrestjs/wiki/Unit-Testing-In-ermrestjs#test-single-spec-file